src/rtc/mosaic_geobursts.py

```diff
@@ -325,9 +532,9 @@ def compute_weighted_mosaic_raster(list_rtc_images, list_nlooks, geo_filename,
 
 
 
-def compute_weighted_mosaic_raster_single_band(
-        list_rtc_images, list_nlooks, output_file_list, geogrid_in=None,
-        verbose=True):
+def mosaic_multiple_output_files(
+        list_rtc_images, list_nlooks, output_file_list, mosaic_mode,
+        scratch_dir='', geogrid_in=None, temp_files_list=None, verbose=True):
     '''
     Mosaic the snapped S1 geobursts
```

Comment:
Please explicitly mention about the difference between this function and `mosaic_single_output_file`